### PR TITLE
Osheenkhare/people picker

### PR DESCRIPTION
**Pull request by osheenkhare**
Added app-people-picker with Code / Screenshots / Docs and Readme.

Also added the header in Readme

### CODE REVIEW

**Comment by Wajeed-msft** [comment]
@osheenkhare - There is an existing sample for People picker linked from the [People Picker in Adaptive Cards](https://docs.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/people-picker?tabs=desktop#code-sample) documentation.  If there are additional scenarios covered then should we merge it to same sample?

**Comment by osheenkhare** [comment]
Hi @Wajeed-msft, this sample actually covers a different use case including Task Module / Adaptive Card and Message Extension with people picker

File: samples/app-people-picker/csharp/Source/Microsoft.Teams.Samples.PeoplePicker/Resources/BotAdaptiveCardTemplate.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    {
+      "type": "TextBlock",
+      "text": "Conversation will be created between the selected users.",
+      "wrap": true
+    },
+    {
+      "type": "TextBlock",
+      "size": "Medium",
+      "weight": "Bolder",
+      "text": "Enter Name of Group Chat"
```
Comment (aosolis) [inline samples/app-people-picker/csharp/Source/Microsoft.Teams.Samples.PeoplePicker/Resources/BotAdaptiveCardTemplate.json:13]: Lets make label text "Sentence case" please

File: samples/app-people-picker/csharp/Source/Microsoft.Teams.Samples.PeoplePicker/Resources/BotAdaptiveCardTemplate.json [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      "type": "TextBlock",
+      "size": "Medium",
+      "weight": "Bolder",
+      "text": "Select Context"
+    },
+    {
+      "type": "Input.ChoiceSet",
+      "choices": [
+        {
+          "title": "CurrentContext",
```
Comment (aosolis) [inline samples/app-people-picker/csharp/Source/Microsoft.Teams.Samples.PeoplePicker/Resources/BotAdaptiveCardTemplate.json:41]: Let's change these titles to more descriptive (keep value the same so code can stay unchanged)

File: samples/app-people-picker/csharp/Source/Microsoft.Teams.Samples.PeoplePicker/Resources/BotAdaptiveCardTemplate.json [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+          "value": "CurrentContext"
+        },
+        {
+          "title": "OrgWide",
+          "value": "OrgWide"
+        }
+      ],
+      "value": "CurrentContext",
+      "id" :  "choice"
```
Comment (aosolis) [inline samples/app-people-picker/csharp/Source/Microsoft.Teams.Samples.PeoplePicker/Resources/BotAdaptiveCardTemplate.json:50]: mark all required field as isRequired=true

**Comment by osheenkhare** [comment]
Fixed Adaptive Card fields

**Pull request closed without merging**